app/src/main/res/values-in/strings.xml

```diff
@@ -12,34 +12,36 @@
     <string name="safetynet">SafetyNet</string>
 
     <!--Home-->
-    <string name="no_connection">Koneksi tidak tersedia</string>
+    <string name="no_connection">Tidak tersedia koneksi internet</string>
```

Comment:
I'll rather have "Koneksi internet tidak tersedia" than "Tidak tersedia koneksi internet", but "internet" here is already an implication.